gitops/argocd/tektoncd/openshift-operator.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: openshift-pipelines-operator
+  namespace: openshift-operators
```

Comment:
No action: I would prefer the operators to be installed in their own namespaces (still separated from operands) rather than openshift-operators. It would require an OperatorGroup first to be created. It would however bring a few benefits:
- better segregation
- dedicated InstallPlan, which provides for instance the possibility to have different upgrade strategy by operator. This is really useful when one runs into troubles.

This can be addressed later on.